api/chalicelib/core/health.py

```diff
@@ -147,7 +147,10 @@ def __check_SSL(*_):
         }
     }
     try:
-        requests.get(config("SITE_URL"), verify=True, allow_redirects=True)
+        if(config("is_dns_public")==True):
```

Comment:
We don't need this block, as there is already an env variable available to skip ssl check
```
SKIP_H_SSL: true
```